### ISSUE TITLE
source install bug (on MacOS arm64) - No such file or directory: 'src/esig/VERSION'

### ISSUE THREAD

**Issue by tarpas** [datasig-ac-uk/esig#127]
I'm getting the error for pip install esig (on a platform which doesn't have wheels)
```
(.venv) ➜  tmpesig pip install esig       
Collecting esig
  Using cached esig-0.9.8.1.tar.gz (202 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  error: subprocess-exited-with-error
  
  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [16 lines of output]
      Traceback (most recent call last):
        File "/Users/tibor/tmonworkspace/tmpesig/.venv/lib/python3.10/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 363, in <module>
          main()
        File "/Users/tibor/tmonworkspace/tmpesig/.venv/lib/python3.10/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 345, in main
          json_out['return_val'] = hook(**hook_input['kwargs'])
        File "/Users/tibor/tmonworkspace/tmpesig/.venv/lib/python3.10/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 130, in get_requires_for_build_wheel
          return hook(config_settings)
        File "/private/var/folders/yv/_xywxyy14q1f0bbj8c64v5_80000gp/T/pip-build-env-p3r7o530/overlay/lib/python3.10/site-packages/setuptools/build_meta.py", line 177, in get_requires_for_build_wheel
          return self._get_build_requires(
        File "/private/var/folders/yv/_xywxyy14q1f0bbj8c64v5_80000gp/T/pip-build-env-p3r7o530/overlay/lib/python3.10/site-packages/setuptools/build_meta.py", line 159, in _get_build_requires
          self.run_setup()
        File "/private/var/folders/yv/_xywxyy14q1f0bbj8c64v5_80000gp/T/pip-build-env-p3r7o530/overlay/lib/python3.10/site-packages/setuptools/build_meta.py", line 174, in run_setup
          exec(compile(code, __file__, 'exec'), locals())
        File "setup.py", line 118, in <module>
          with io.open("src/esig/VERSION", "rt") as fp:
      FileNotFoundError: [Errno 2] No such file or directory: 'src/esig/VERSION'
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.
```

**Comment by inakleinbottle** [comment]
Thanks for raising an issue. What platform are you building esig for? This issue hopefully is easily fixed, but building esig is notoriously difficult because of its dependencies.

**Comment by tarpas** [comment]
MacOs on Apple Silicon.

I assumed it's this commit: https://github.com/datasig-ac-uk/esig/commit/0fae49ebcab27026c1b86648b1cfbad016412a8e

Because previous versions don't have this specific problem. 

As for difficulty of building esig, yes I wasn't able to build or install it on my machine, so I expect that even after fixing this issue I'll not be able to build it.

**Comment by inakleinbottle** [comment]
Ok this is caused by a problem with the MANIFEST.in file not having been updated for the newer version of esig. I've fixed this in #128 along with another problem.
